src/dune_pkg/opam_solver.ml

```diff
@@ -237,6 +237,20 @@ let opam_command_to_string_debug (args, _filter_opt) =
   |> String.concat ~sep:" "
 ;;
 
+let package_variable_of_opam_ident package ident =
+  let err_not_supported var package =
```

Comment:
Any reason why this is a function? I think it can just be inlined.

Reply:
Sure I'll do that.